---
title: "MYC overrides HIF to regulate proliferating primary cell metabolism in hypoxia"

author:
- Courtney A. Copeland: 
    institute: bwh, hms
- Benjamin A. Olenchock: 
    institute: bwh, hms
- David R. Ziehr:
    institute: bwh, mgh, hms
- Sarah McGarrity: 
    institute: bwh, hms, ui
- Kevin Leahy: 
    institute: bwh, hms
- Jamey D. Young: 
    institute: vu
- Joseph Loscalzo: 
    institute: bwh, hms
- William M. Oldham:
    institute: bwh, hms
    correspondence: true
    email: woldham@bwh.harvard.edu
    
institute:
- bwh: Department of Medicine, Brigham and Women's Hospital, Boston, MA 02115, U.S.A.
- hms: Department of Medicine, Harvard Medical School, Boston, MA 02115, U.S.A.
- mgh: Department of Medicine, Massachusetts General Hospital, Boston, MA, 02114 U.S.A.
- ui: Center for Systems Biology, School of Health Sciences, University of Iceland, Reykjavik, Iceland
- vu: Departments of Chemical & Biomolecular Engineering and Molecular Physiology & Biophysics, Vanderbilt University, Nashville, TN 37240, U.S.A.

bibliography:
- library.json
- packages.bib
csl: cell-metabolism.csl
link-citations: yes

output:
  bookdown::word_document2:
    reference_docx: !expr system.file("manuscript", "template.docx", package = "Copeland.2022.hypoxia.flux")
    df_print: kable
    fig_caption: true
    number_sections: false
    pandoc_args:
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
      - '--lua-filter=pagebreak.lua'
      # - '--lua-filter=multiple-bibliographies.lua'

compact-title: true
geometry: margin = 0.75in
fontsize: 11pt
mainfont: Calibri
graphics: yes
---

```{r setup, include = FALSE}
# library(Copeland.2022.hypoxia.flux)
devtools::load_all()

knitr::opts_chunk$set(
  echo = FALSE, 
  warning = FALSE, 
  message = FALSE, 
  out.width = "100%", 
  out.extra = ""
)

rbbt::bbt_update_bib(
  path = "manuscript/manuscript.Rmd",
  path_bib = "manuscript/library.json"
)

options(
  tinytex.verbose = TRUE, 
  knitr.graphics.rel_path = FALSE
)

# targets::tar_load(
#   c(
#     m1_figure, 
#   )
# )
```

# Abstract

# Keywords

/ / 

\newpage

# Introduction

Cellular responses to hypoxia propel many physiologic and pathologic activities from wound healing and angiogenesis to vascular remodeling and fibrosis [@semenza2012b; @lee2019]. These activities require cells to continue energetically demanding tasks, such as macromolecular biosynthesis and proliferation, despite limited oxygen availability. Since respiration is the most efficient way for cells to produce energy, cell metabolism must adapt to meet energetic needs when oxygen supply is limiting. Understanding how these metabolic adaptations sustain critical cellular processes in hypoxia is fundamentally important to our understanding of human health and disease. 

Cells typically respond to hypoxia by shifting energy production away from respiration and toward glycolysis. This response is mediated primarily by stabilization of the hypoxia-inducible transcription factor 1α (HIF-1α). HIF-1α activates the transcription of glucose transporters, glycolytic enzymes, lactate dehydrogenase, and pyruvate dehydrogenase kinase, while decreasing the expression of tricarboxylic acid (TCA) cycle and electron transport chain enzymes [@semenza2012b; @lee2020b]. Although HIF-1α is constitutively expressed, it is hydroxylated by prolyl hydroxylase enzymes (PHDs) in normoxia and targeted for proteasomal degradation. PHDs are the principal oxygen sensors in metazoan cells [@kaelin2008a]. PHDs are α-ketoglutarate-dependent dioxygenase enzymes that require molecular oxygen for their enzymatic activity. When oxygen tension falls, PHD activity decreases, leading to HIF-1α stabilization and activation of its associated transcriptional program. Overall, this transcriptional program should increase glycolytic capacity and divert glucose-derived pyruvate from oxidative phosphorylation toward lactate fermentation to maintain glycolytic ATP production. 

In addition to metabolic changes designed to maintain energy supply, hypoxic cells also reduce energy demands through down-regulation of Na^+^/K^+^-ATPase, slowing protein translation, and attenuating cell proliferation [@wheaton2011; @hubbi2015]. In particular, HIF-1α decreases cell proliferation by activating cyclin-dependent kinase inhibitor expression, inhibiting cell cycle checkpoint progression [@gardner2001], and antagonizing pro-proliferative MYC signaling [@koshiji2004]. Despite these canonical effects of HIF-1α activation, there are many examples where cells continue to proliferate despite hypoxic stress, including cancer cells, stem cells, and lung vascular cells [@hubbi2015]. How these cells meet the metabolic needs of sustained proliferation in hypoxia and how these adaptations are regulated are an active area of investigation [@jain2020; @oldham2015; @lee2020b]. Since hypoxia is a prominent feature of cancer biology as tumor growth outstrips blood supply, most detailed metabolic studies of hypoxic cell metabolism have used tumor cell models, yielding important insights into the metabolic pathobiology of cancer [@wise2011; @metallo2011; @melendez-rodriguez2019; @garcia-bermudez2018; @lee2019a; @jiang2016]. For example, stable isotope tracing and metabolic flux analyses identified a critical role for reductive carboxylation of glutamine-derived α-ketoglutarate for lipid biosynthesis in supporting tumor growth [@metallo2011; @gameiro2013a; @scott2011; @wise2011], and metabolomic studies identified aspartate as a limiting metabolite for cancer cell proliferation under hypoxia [@garcia-bermudez2018]. By contrast, comparatively little is known about metabolic adaptations of primary cells to hypoxia. Indeed, the importance of reductive carboxylation or aspartate biosynthesis remains to be elucidated in these cells. A more complete understanding of primary cell metabolic adaptation to hypoxia would provide an important context for understanding how metabolic reprogramming supports normal cellular responses to hypoxia, how these responses may be (mal)adaptive in a variety of disease contexts, and how the hypoxia metabolic program in primary cells differs from that observed in cancer cells. 

To address these questions, we have developed models of bioenergetic carbon flux in human lung fibroblasts (LFs) and pulmonary artery smooth muscle cells (PASMCs) cultured in 21% or 0.5% oxygen. These cells may be exposed to a wide range of oxygen concentrations *in vivo*, continue to proliferate despite hypoxic culture conditions *in vitro*, and play important roles in the pathology of non-cancerous diseases in which tissue hypoxia is a prominent feature. We found that hypoxia fails to increase glycolysis in these primary cells despite robust up-regulation of the HIF-1α transcriptional program. In normoxia, HIF-1α stabilization by the PHD inhibitor molidustat (BAY-85-3934, "BAY") [@flamme2014] did increase glycolysis and lactate efflux; however, hypoxia blocked this response. These findings suggested the existence of important hypoxia-dependent regulatory mechanisms that override the metabolic consequences of HIF-1α-dependent up-regulation of glycolytic gene expression in human primary cells. Transcriptomic profiling suggested a critical role for the transcription factor MYC in the adaptive response to hypoxia. Using knockdown and overexpression approaches, we demonstrate that MYC attenuates HIF-driven glycolysis in hypoxia and following BAY treatment.

\newpage

# Results

## Hypoxia uncouples glycolytic gene expression from glycolytic metabolic flux

The goal of this study was to characterize hypoxia-induced metabolic changes in proliferating primary LFs and PASMCs. To accomplish this goal, we adopted a metabolic flux analysis (MFA) technique that enabled us to link intracellular metabolic fluxes to cell proliferation rates. MFA fits cell proliferation rate, extracellular flux measurements, and ^13^C intracellular isotope labeling patterns to a model of cell metabolism [@antoniewicz2018]. This analysis reconstructs comprehensive flux maps that depict the flow of carbon from extracellular substrates, throughout intracellular metabolism, and into cell biomass and metabolic by-products [@young2014]. These models assume that cells are at a metabolic pseudo-steady state over the experimental time course [@buescher2015]. Exponential growth phase is thought to reflect metabolic pseudo-steady state, because cells in the culture steadily divide at their maximal condition-specific rate, as long as nutrient supply does not become limiting [@buescher2015; @ahn2011]. Thus, we first set out to define experimental conditions to capture exponential growth phase in normoxic and hypoxic cultures. 

Cells were seeded and placed into hypoxia for 24 h prior to sample collection to provide adequate time for activation of the hypoxia-dependent transcriptional program (**Fig \@ref(fig:m1)A**). We selected 0.5% oxygen for hypoxia as this level yielded the most reproducible phenotypic differences compared to 21% oxygen culture while being physiologically relevant and above the K~M~ of cytochrome *c* oxidase (electron transport chain complex IV) for oxygen [@lee2020b; @wenger2015]. From this starting point, we identified the optimal cell seeding density and time course to capture exponential cell growth (**Fig \@ref(fig:m1)B**). LFs cultured in 0.5% oxygen grew more slowly than LFs cultured in 21% oxygen (**Fig \@ref(fig:m1)C**), but slower growth was not associated with decreased cell viability (**Fig S1A**). As anticipated, hypoxic cells demonstrated robust stabilization of HIF-1α protein associated with up-regulation of downstream targets, such as glucose transporter 1 (GLUT1) and lactate dehydrogenase A (LDHA) (**Figs \@ref(fig:m1)D-H**). These changes persisted for the duration of the experimental time course.

Having identified experimental conditions for exponential growth, we next determined extracellular fluxes of glucose (GLC), lactate (LAC), pyruvate (PYR), and amino acids (**Figs \@ref(fig:m1)I-J**). Flux calculations incorporated changes in cell number, extracellular metabolite concentrations, metabolite degradation rates, and medium evaporation over time [@murphy2013a] (**Figure S1**). Interestingly, while we observed a modest increase in glucose uptake, we found that hypoxia actually decreased lactate efflux (**Figs \@ref(fig:m1)I**). This decrease in lactate efflux occurred despite activation of the HIF-1α transcriptional program as reflected by increased expression of GLUT1 and LDHA. To test if more severe hypoxia would augment glycolysis, we culture cells in 0.2% ambient oxygen (**Fig S2**). These results were similar to 0.5% oxygen culture. We observed no change in glucose or lactate fluxes. To test if this unexpected response was unique to LFs, we next studied PASMCs under 0.5% oxygen conditions (**Fig S3**). PASMCs grew faster than LFs, and so samples were collected every 12 h for 48 h for these cells. Again, similar to LFs at 0.5% and 0.2% oxygen, we observed no change in glucose uptake and reduced lactate efflux in PASMCs regardless of HIF-1α stabilization. Together, these data suggest that hypoxia uncouples HIF target gene expression and glycolytic flux in proliferating primary cells. 

## HIF stabilization in normoxia does increase glycolysis

Given that hypoxia exposure did not increase glycolysis in LFs, we next wanted to determine how these cells responded to HIF-1α stabilization in normoxia. To accomplish this, LFs were treated with the PHD inhibitor molidustat (BAY) using a similar time course as our hypoxia experiments. Cells were treated with BAY for 24 h to activate the HIF transcriptional program prior to sample collection (**Fig \@ref(fig:m2)A**). Similar to hypoxia treatment, BAY decreased cell growth rate (**Figs \@ref(fig:m2)B-C**) and activated the HIF-1α transcriptional program (**Figs \@ref(fig:m1)D-H**). Compared to hypoxia, BAY treatment resulted in a similar activation of HIF-1α target gene transcription and protein expression. Unlike hypoxia, HIF-1α stabilization in normoxia markedly increased glucose uptake and lactate efflux (**Fig \@ref(fig:m1)I**), as expected based on the pattern of glycolytic gene expression that we observed. Interestingly, although hypoxia and BAY treatments resulted in similar increases in HIF-1α, GLUT1, and LDHA, the glycolytic response was markedly different. 

## Extracellular flux measurements highlight subtle differences between hypoxia and BAY

In addition to glucose and lactate, we also determined the extracellular fluxes of pyruvate and amino acids (**Figs \@ref(fig:m1)J, \@ref(fig:m2)J**). Overall, changes were modest, with hypoxia generally decreasing the fluxes of all measured metabolites. Notably, a substantial decrease in glutamine consumption was observed in LFs cultured in 0.5% oxygen. This observation is in contrast to previous studies in cancer cell metabolism demonstrating increased glutamine uptake as a key feature of the metabolic response to hypoxia [@metallo2011; @wise2011; @gameiro2013]. Similar patterns were observed in 0.2% oxygen (**Figure S2**). In PASMCs, glutamine uptake did increase, as did the uptake of branched-chain amino acids and arginine (**Figure S3**), highlighting differential responses of these mesenchymal cells to hypoxia. Comparatively modest effects of BAY on amino acid fluxes were observed as compared to 0.5% oxygen culture conditions (**Figure \@ref(fig:m1)J**). BAY treatment elicited different responses to hypoxia with relative preservation of glutamine and asparagine uptake and alanine and glutamate efflux compared to hypoxia.

## Stable isotope labeling 


<!-- discuss LF v. PASMC -->



\newpage

# Discussion

\newpage

# Materials and Methods

## Data Availability

\newpage

# Acknowledgements

# Author Contributions

# Disclosures and competing interests

\newpage

# References

::: {#refs} 
:::

\newpage

# Tables

\newpage

# Figure Legends

(ref:m1) **Caption.** Info.

```{r}
figs <- 
  list.files(
    system.file(
      "manuscript/figures", 
      package = "package"
    ), 
    pattern = "m\\d{1}\\.png"
  ) %>% 
  rlang::set_names(stringr::str_extract(., "^.*(?=\\.png)"))
```

```{r results = "asis"}
out <- 
  purrr::imap(
    figs, 
    ~ cat("Figure ", stringr::str_extract(.y, "\\d"), ": (ref:", .y, ")\n\n", sep = "")
  )
```

\newpage

# Figures

```{r m1, fig.cap = "(ref:m1)"}
# knitr::include_graphics(
#   system.file(
#     "manuscript/figures/m1.png",
#     package = "package"
#   )
# )
```

\newpage
